demos/array-shell-sort/README.md

```diff
@@ -0,0 +1,41 @@
+# Merge Sort Demo
```

Comment:
probably a minor typo or copy-paste thing, but it says merge sort here ^

Reply:
How embarrassing! Resolved with last push.